db/controllers/getTopics.controller.js

```diff
@@ -0,0 +1,13 @@
+const { selectTopics } = require("../models/getTopics.model.js");
+console.log('in the controller');
+exports.getTopics = (req, res, next) => {
+//  const query = req.query;
+  //console.log(query.sort_by, query);
+  selectTopics()
+    .then((arrayOfTopics) => {
+      res.status(200).send({ topics: arrayOfTopics });
+    })
+    .catch((err) => {
```

Comment:
We only need to add a catch block when we are manually throwing errors (for example using Promise.reject() in the model).  since we are not doing this in selectTopics, we don't have to have a catch block here.